surefire-booter/src/main/java/org/apache/maven/surefire/booter/ForkedBooter.java

```diff
@@ -236,13 +236,17 @@ private static void exit( int returnCode, Shutdown shutdownType, CommandReader r
         switch ( shutdownType )
         {
             case KILL:
+                System.out.close();
```

Comment:
The logs show me that `Process.exec()` is blocked with maybe read() method in another Thread, and maybe a kind of deadlock which means using error stream might be a good try.